packages/babel-plugin-transform-for-of/src/index.js

```diff
@@ -107,49 +161,6 @@ export default function(api, options) {
     }
   }
 
-  return {
```

Comment:
none of the code here is any different right? (just saying this part of the diff isn't necessary as clarification)